tests/test_capacity.py

```diff
@@ -76,21 +76,24 @@ def test_equality_to_zero():
 def test_new_style_formatting():
     assert "{0}".format(3 * GiB) == str(3 * GiB)
     assert "{0!r}".format(3 * GiB) == repr(3 * GiB)
-    assert "{0:GiB}".format(3 * GiB) == "3"
+    assert "{0:GiB}".format(3 * GiB) == "3.0"
```

Comment:
I don't think this is reasonable. capacities obtained from integer multiplication shouldn't format as floats...